index.html

```diff
@@ -53,6 +53,40 @@ <h2 class="connect">LET’S CONNECT</h2>
     </div>
 
   </section>
+  
+  <section class="work-section">
+    <div class="card_work">
+      <img class="snapshot" src="images/Snapshoot Portfolio.png" alt="">
+      <div class="left_block">
+        <h1 class="tonic-text">Tonic</h1>
+        <div class="frame-2">
+          <h4 class="canopy">CANOPY</h4>
+          <img class="count" src="images/Counter.png" alt="">
+          <h4 class="role">Back End Dev</h4>
+          <img class="count" src="images/Counter.png" alt="">
+          <h4 class="year">2015</h4>
+        </div>
+        <h3 class="daily_text">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</h3>
```

Comment:
Since this is a paragraph kindly consider using the p tag instead of the h3 tag. :+1: